CMS/settings.py

```diff
@@ -145,6 +145,17 @@
     'DEFAULT_AUTHENTICATION_CLASSES': (
         'rest_framework_simplejwt.authentication.JWTAuthentication',
     ),
+    'DEFAULT_THROTTLE_CLASSES': [
+        'rest_framework.throttling.UserRateThrottle',
+        'blog.throttles.BlogRateThrottle',
+        'accounts.throttles.AccountsRateThrottle'
+
+    ],
+    'DEFAULT_THROTTLE_RATES': {
+        'user': '200/day',
+        'blog': '100/day',
+        'accounts': '50/day'
```

Comment:
This will be of much interest to the team working on the logs. 
Maybe I loop them in on this. 
@mamu234 @AmbroseOtundo  @JosephThuku

Reply:
So are these requests to our API? that is for each user.

Reply:
intresting. so you are limiting the number of times the api can be called by a user, or an account.
I would like to see how it works on our next meeting

Reply:
The limits for verified users are different from unverified, and for the unverified there is a higher limit when they visit the blog endpoints, but slightly less for the account endpoints. I decided this way since the blogs may be viewed more times. I'll explain further in the next meeting.